### PR TITLE
minor content tweaks to header and footer

### PR DESCRIPTION
**Pull request by ajefts**
@parthshah Just doing some minor tweaks to get familiar with working with the code base and see how we work with the env...

### CODE REVIEW

File: src/components/TopBar/TopBar.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ class TopBar extends Component {
 
   render() {
     const username = this.props.username
-    const userImage = this.props.userImage
+    const userImage = this.props.photoUrl
```
Comment (parthshah) [inline src/components/TopBar/TopBar.jsx]: Are you sure this works? Layout.jsx is providing props - username, userImage & domain. Where do you get photoUrl from?

Comment (ajefts) [inline src/components/TopBar/TopBar.jsx]: @parthshah I'm not actually.  userImage didn't seem to be working.  I noticed photoUrl in a couple other places for the user image.  When I changed it to photoUrl, it actually did have the effect of using the silhouette, but looks like it's not pulling in the image that's in my profile.

So, you're thinking props.userImage should be populated correctly?

Comment (ajefts) [inline src/components/TopBar/TopBar.jsx]: fyi @parthshah I switched it back to userImage. Looks like that is returning the user_id, as  opposed to the image...

**Comment by parthshah** [comment]
@ajefts i'm closing this PR... the branches have diverted quite a bit. If we still need these changes it would be best to create a new PR against latest Dev.

**Comment by ajefts** [comment]
no problem at all @parthshah

**Comment by vic-appirio** [comment]
Deleted it - it is quite old at this point

**Pull request closed without merging**